File: Lecture2/Celsius2Fahrenheit.c
#include <stdio.h>

int main(void) {
	
	double celsius, fahrenheit;
	
	printf("Celsius : ");
	scanf("%lf", &celsius);
	
	fahrenheit = ( 9.0 / 5 ) * celsius + 32;
	
	printf("%lf C == %lf F", celsius, fahrenheit);
	
	return 0;

}
	

File: Lecture2/Celsius2Fahrenheit.java
import java.util.Scanner;

public class Celsius2Fahrenheit {
	
	public static void main(String[] args) {
		
		Scanner scan = new Scanner(System.in);
		
		double celsius, fahrenheit;
		
		System.out.print( "Celsius : " );
		celsius = scan.nextDouble();
		
		fahrenheit = ( 9.0 / 5 ) * celsius + 32;

		System.out.println( celsius + " C == " + fahrenheit + " F");
	}
}
		